azure-pipelines.yml

```diff
@@ -21,6 +22,8 @@ jobs:
     service_connection_nuget_org: $(Endjin_Service_Connection_NuGet_Org)
     service_connection_github: $(Endjin_Service_Connection_GitHub)
     solution_to_build: $(Endjin_Solution_To_Build)
+    netSdkVersion: '6.x'
+    nupkgversionToolPath: 'dotnet $(Build.SourcesDirectory)/Solutions/Endjin.SemVer.DotNetApi.Cli/bin/$(BuildConfiguration)/net6.0/nupkgversion.dll'
```

Comment:
I don't understand what's happening here, so I think it would be really good to add a comment to the yaml.